src/libOpenImageIO/color_ocio.cpp

```diff
@@ -698,26 +726,36 @@ void
 ColorConfig::Impl::identify_builtin_equivalents()
 {
 #if OCIO_VERSION_HEX >= MAKE_OCIO_VERSION_HEX(2, 3, 0)
+    Timer timer;
```

Comment:
`OIIO_MAYBE_UNUSED Timer timer;`

Reply:
Is that needed? I'm not seeing any build errors on CI. I think that the constructor of the timer itself "does something", so it's not unused.

Reply:
Unsure if it's really needed but another method a little below this used that pattern.